src/components/sync-dialog/use-copy.tsx

```diff
@@ -0,0 +1,62 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { useMemo } from 'react';
+
+export const useCopy = ( type: 'pull' | 'push' ) => {
```

Comment:
Let's rename this hook to something like `useSyncTexts`. Now that we are exporting the hook, it may cause some confusion.

```suggestion
export const useSyncTexts = ( type: 'pull' | 'push' ) => {
```

Reply:
If we are going to extract it to a separate file, then we should create `tree-view` folder and keep it inside of this folder.
But even w/o it, I would move this changes to a separate PR, to keep this PR simpel and focused on specific changes.

Reply:
Ideally, we refactor the code in separate PRs, but it’s fine to clean the code as we progress. So, there’s no need to undo these changes. I’ll defer the decision to @bcotrim .

Reply:
I agree we could probably do a big code cleanup before completing the project.
To make things easier to review, I reverted the changes in https://github.com/Automattic/studio/pull/1484/commits/7dc7227152c17bbd1a3af6672e026c5397726be4

Reply:
Thank you, it's really much easier to review the PR now

Reply:
> If we are going to extract it to a separate file, then we should create tree-view folder and keep it inside of this folder, instead of sync-dialog (a level above).

@bcotrim BTW, regarding this ^^^ - my bad, I mixed up `tree-view` and `sync-dialog`. The changes were good 👍 But still, let's do it separatelly.

Reply:
I created a issue to track it: STU-580